rero_ils/modules/entities/local_entities/api.py

```diff
@@ -57,7 +59,7 @@ class Meta:
         default_filter = None
 
 
-class LocalEntity(IlsRecord):
+class LocalEntity(IlsRecord, ABC):
```

Comment:
> @jma told me that "abstract" doesn't really exists in Python world... The paradigm is to use "Mixin". Am I correct ? But in this specific case.... I can choose myself the best solution.

Abstract classes, Mixins and multiple inheritance are different. A mixin should be an abstract class in the sense that they are not supposed to be instantiate. Which is not the case in the usual multiple inheritance. It can be seen as similar than interfaces in other languages. Abstract classes are not explicit in python but can be implemented with ABC or just by raise the `NotImplemented` exception.

Reply:
thx @jma. @lauren-d You can forget my comment ;-)